less/_mixins.less

```diff
@@ -48,18 +39,23 @@
 
 // hidden (accessible by screen readers)
 .hidden {
```

Comment:
Right, saw that and it makes me happy. I think the names should also be changed to reflect more of what they do. Hidden from everything vs hidden from a screen reader, vs hidden from desktop but not a screenreader vs Godzilla vs whatever…